azurelinuxagent/common/utils/networkutil.py

```diff
@@ -120,6 +120,7 @@ class AddFirewallRules(object):
     """
 
     __ACCEPT_COMMAND = "-A"
+    __INSERT_COMMAND = "-I"
```

Comment:
Can you add a small comment here on the difference between `-A` and `-I`

Reply:
Addressed